content/guides/tooling/sdks/index.md

```diff
@@ -91,6 +56,32 @@ build apps on Android.
 Refer to [this][android-docs] documentation for more details.
 </Message>
 
+## Next Generation SDKs
+
+<Message type='warning'>
+  As of September 17, 2025 Box Next Generation SDKs are no longer supported as separate artifacts.
+
+  Don't worry, your existing code will continue to work without changes. You can still use your applications based on Box Next Generation SDKs with no impact, but you won't receive new features, updates, or bug fixes.
+
+  In keeping with industry best practices, we are consolidating the Box Next Generation SDKs and Box core SDKs into a single package for each programming language. This makes migration efforts much easier and allows to seamlessly add new capabilities to existing applications still powered by the manually maintained Box core SDKs.
+
+  All future development, including new features and updates for the Next Generation SDKs, will be delivered through Box core SDKs starting with version `v10`. Currently, `v10` available as a separate branch.
+
+  For more details, see our [SDK versioning strategy document][versioning].
+</Message>
+
+The latest generation Box Python SDK, Box TypeScript
```

Comment:
Not consistant naming - put Box before .NET SDK and Swift SDK as well

Reply:
Done